soql-analyzer/src/main/scala/com/socrata/soql/functions/Function.scala

```diff
@@ -24,7 +27,9 @@ case class Function[Type](identity: String,
                           parameters: Seq[TypeLike[Type]],
                           repeated: Seq[TypeLike[Type]],
                           result: TypeLike[Type],
-                          isAggregate: Boolean = false)  {
+                          isAggregate: Boolean,
```

Comment:
was losing the default value here on purpose?

Reply:
yep, you can't have default values and variadic arguments in the same signature.